code/game/objects/items/weapons/tools/wirecutters.dm

```diff
@@ -17,29 +15,32 @@
 //Better and more flexible than most improvised tools, but more bulky and annoying to make
 /obj/item/weapon/tool/wirecutters/improvised
 	name = "wiremanglers"
-	desc = "An improvised monstrosity made of bent rods which can sometimes be used to snip things."
+	desc = "An improvised monstrosity made of bent rods which can sometimes be used to snip things. Could serve you well if you stuff it with enough tool mods."
 	icon_state = "impro_cutter"
 	w_class = ITEM_SIZE_NORMAL
 	force = WEAPON_FORCE_NORMAL
 	tool_qualities = list(QUALITY_WIRE_CUTTING = 20, QUALITY_RETRACTING = 10, QUALITY_BONE_SETTING = 10)
 	degradation = 1.5
+	max_upgrades = 5 //all makeshift tools get more mods to make them actually viable for mid-late game
 
 /obj/item/weapon/tool/wirecutters/armature
 	name = "armature cutter"
 	desc = "Bigger brother of wirecutter. Can't do much in terms of emergency surgery, but does its main job better."
 	icon_state = "arm-cutter"
+	w_class = ITEM_SIZE_NORMAL
 	force = WEAPON_FORCE_NORMAL
 	matter = list(MATERIAL_STEEL = 4, MATERIAL_PLASTEEL = 1, MATERIAL_PLASTIC = 1)
 	tool_qualities = list(QUALITY_WIRE_CUTTING = 40, QUALITY_CUTTING = 30)
 	degradation = 0.7
 	max_upgrades = 4
 
-/obj/item/weapon/tool/wirecutters/pliers
+/obj/item/weapon/tool/wirecutters/pliers //hybrid of wirecutters, wrench and cautery
 	name = "pliers"
-	desc = "Useful for pinching, clamping and occasional bolt turning."
+	desc = "Multitool from the world of maintenance. Useful for pinching, clamping and occasional bolt turning."
```

Comment:
"A multitool from the world of maintenance. Useful for pinching, clamping, and occasionally bolt turning."